tasks/Task.cpp

```diff
@@ -395,7 +396,8 @@ MessageStatus Task::checkMessageStatus()
         if(base::Time::now() - last_im_sent > _timeout_delivery_report.get())
         {
             exception(MISSING_DELIVERY_REPORT);
-            throw runtime_error("Timeout while wait for a delivery report.");
+            throw runtime_error("Timeout while wait for a delivery report. Timeout of "
+                    + boost::lexical_cast<string>(_timeout_delivery_report.get()) + "ms" );
```

Comment:
I just took a look on how it was done on [iodrivers_base](https://github.com/rock-drivers/drivers-iodrivers_base/blob/master/src/Driver.cpp#L740). 
Not sure which one to use.